python/lsst/ip/diffim/detectAndMeasure.py

```diff
@@ -681,7 +695,31 @@ def _runStreakMasking(self, maskedImage):
             ``modelMaximum`` : `np.ndarray`
                 Peak value of the fit line profile.
         """
-        streaks = self.maskStreaks.run(maskedImage)
+        maskedImage = difference.maskedImage
+        # Bin the diffim to enhance low surface brightness streaks
+        binnedMaskedImage = afwMath.binImage(maskedImage,
+                                             self.config.streakBinFactor,
+                                             self.config.streakBinFactor)
+        binnedExposure = afwImage.ExposureF(binnedMaskedImage.getBBox())
+        binnedExposure.setMaskedImage(binnedMaskedImage)
+        binnedExposure.setPsf(difference.psf)  # exposure must have a PSF
```

Comment:
So I have been informed that there is a possibility that you can reach this point in the pipeline and an exposure might not have a PSF. How does that get handled in this situation? Does this automatically fail out gracefully or do we need to add some error handling just in case.

Reply:
Thank you for thinking of this — indeed this was a past problem, so I verified that the `_prepareInputs` call in the `run` method of detectAndMeasure will raise well before we get to streak masking if the PSF is un-compute-shape-able.